src/etc/inc/ipsec.inc

```diff
@@ -1895,7 +1899,7 @@ function ipsec_setup_proposal_entry(& $ph2ent, & $algo_arr, $ealg_id, $keylen) {
 	/* If multiple hash algorithms are present, loop through and add them all. */
 	if (!empty($ph2ent['hash-algorithm-option']) && is_array($ph2ent['hash-algorithm-option'])) {
 		foreach ($ph2ent['hash-algorithm-option'] as $halgo) {
-			$proposal[] = ipsec_setup_proposal_algo($ealg_id, $keylen, $halgo, $ph2ent['pfsgroup']);
+			$proposal[] = ipsec_setup_proposal_algo($ealg_id, $keylen, $halgo, False, $ph2ent['pfsgroup']);
```

Comment:
This should either be `false` or `FALSE`, preferably `false` since that's what is used elsewhere in the file.

Reply:
done